emfed.ts

```diff
@@ -184,5 +184,22 @@ async function loadToots(element: Element) {
   }
 }
 
-// Automatically transform links with a special class.
-document.querySelectorAll("a.mastodon-feed").forEach(loadToots);
+/**
+ * Automatically transform links with a special class.
+ */
+export function loadAll() {
+  document.querySelectorAll("a.mastodon-feed").forEach(loadToots);
+}
+
+// By default, we automatically transform all links. But users can disable
+// this by supplying ?auto=off and instead manually invoking the API to
+// transform specific elements.
+if (
+  !(
+    import.meta &&
+    import.meta.url &&
+    new URL(import.meta.url).searchParams.get("auto") === "off"
```

Comment:
Webpack's compile time tree-shaker is not going to be able to evaluate this condition.

Reply:
Forgive me if I'm misunderstanding something, but my original thought here was that a bundler would not need to evaluate the condition—it would just need to preserve it, so that it evaluates to `false` at run time. Does that make any sense?

Reply:
Oh, that was dumb; I of course was getting the condition backward in my head. We would want this to "default to off" instead of "defaulting to on." Sorry about that.

Reply:
Even if the condition defaults to false, the `new URL(import.meta.url)` code would still need to get executed at runtime (if I understand correctly) which seems janky.